src/components/Calculator.js

```diff
@@ -0,0 +1,90 @@
+import './Calculator.css';
+
+const Calculator = () => (
```

Comment:
- [ ] As I highlighted in the review summary 👏 Please take a while of your precious time to convert the components to class-based components at this stage of the project 😄 🙏

Reply:
Resolved